tests/python/pants_test/backend/python/tasks/test_python_distribution_integration.py

```diff
@@ -66,7 +65,7 @@ def test_invalidation(self):
     with self.mock_buildroot(
         dirs_to_copy=[self.fasthello_install_requires_dir]) as buildroot, buildroot.pushd():
       run_target = lambda: self.run_pants_with_workdir(
-        command=['-ldebug', 'run', fasthello_run],
+        command=['run', fasthello_run],
         workdir=os.path.join(buildroot.new_buildroot, '.pants.d'),
         build_root=buildroot.new_buildroot,
         extra_env={'PEX_VERBOSE': '9'},
```

Comment:
Consider killing these - they were added to debug as I recall.

Reply:
Removed all the `PEX_VERBOSE` from the codebase, including a stray in the testprojects integration test (in 4456105).